Makefile

```diff
@@ -6,26 +6,41 @@ install-frontend:
 install-frontend-ci:
 	cd web && npm ci
 
-.PHONY: lint-frontend
-lint-frontend:
-	cd web && npm run lint
+.PHONY: install-frontend-ci-clean
```

Comment:
I think you might need to add `install-frontend-ci` in this make command: 

```
.PHONY: install-frontend-ci-clean
install-frontend-ci-clean: install-frontend-ci
	cd web && npm cache clean --force
```

Reply:
See logging-view-plugin for reference: https://github.com/openshift/logging-view-plugin/blob/c15865761f06c4e081e51be7477287cbcfd4508a/Makefile#L11C1-L13C35